main.go

```diff
@@ -196,22 +194,29 @@ func CreateExecutionPlan(cfg *Config) ([]string, error) {
 			args = append(args, "--timeout", cfg.Timeout)
 		}
 
-		if cfg.Region != "" {
-			args = append(args, "--region", cfg.Region)
-		}
-
-		for flg, argStr := range cfg.AdditionalFlags {
-			if argStr != "" {
-				args = append(args, fmt.Sprintf("--%s=%s", flg, argStr))
-			} else {
-				args = append(args, fmt.Sprintf("--%s", flg))
-			}
-		}
+	case "update-traffic":
+		args = append(args, "services", "update-traffic")
+		args = append(args, cfg.ServiceName)
 
 	default:
 		return []string{}, fmt.Errorf("action: %s not implemented yet", cfg.Action)
 	}
 
+	args = append(args, "--project", cfg.Project)
+	args = append(args, "--platform", cfg.Runtime)
+
+	if cfg.Region != "" {
+		args = append(args, "--region", cfg.Region)
+	}
+
+	for flg, argStr := range cfg.AdditionalFlags {
+		if argStr != "" {
+			args = append(args, fmt.Sprintf("--%s=%s", flg, argStr))
```

Comment:
Speaking of canary use case, I think this won't work with your `--tags canary=5` example?
Will this have to change to
```
 			args = append(args, flg)
 			args = append(args, argStr)
```
to avoid issues with the double `=`?

Reply:
I just tried it out and it seemed to work fine! I don't know about a general rule, but with gcloud commands I've found `=`s are valid inside flags, even when using it as a delimiter like `--tags=canary=5`.

Reply:
👍